data/mods/Xedra_Evolved/spells/eater_spells.json

```
@@ -0,0 +1,120 @@
+[
+  {
+    "id": "eat_dreamdross",
+    "type": "SPELL",
+    "name": "Eat dreamdross",
+    "description": "You consume dreamdross to regain mana.",
+    "valid_targets": [ "self" ],
+    "min_damage": 75,
+    "damage_increment": 15.0,
+    "max_damage": 100,
+    "max_level": 35,
+    "effect": "recover_energy",
+    "effect_str": "MANA",
+    "shape": "blast",
+    "spell_class": "EATER",
+    "energy_source": "HP",
+    "base_casting_time": 500,
+    "base_energy_cost": 0,
+    "final_energy_cost": 0,
+    "components": "spell_components_dreamdross",
+    "flags": [ "SOMATIC", "VERBAL", "SILENT", "NO_LEGS" ],
+    "difficulty": 1
+  },
+  {
+    "id": "eat_more_dreamdross",
+    "type": "SPELL",
+    "name": "Eat more dreamdross",
+    "description": "You can consume more dreamdross to regain mana in one sitting.",
+    "valid_targets": [ "self" ],
+    "min_damage": 225,
+    "damage_increment": 15.0,
+    "max_damage": 300,
+    "max_level": 35,
+    "effect": "recover_energy",
+    "effect_str": "MANA",
+    "shape": "blast",
+    "spell_class": "EATER",
+    "energy_source": "HP",
+    "base_casting_time": 500,
+    "base_energy_cost": 0,
+    "final_energy_cost": 0,
+    "components": "spell_components_dreamdross2",
+    "flags": [ "SOMATIC", "VERBAL", "SILENT", "NO_LEGS" ],
+    "difficulty": 1
+  },
+  {
+    "id": "eat_dreamdross_ingot",
+    "type": "SPELL",
+    "name": "Eat dreamdross ingots",
+    "description": "You can consume ingots of dreamdross to regain mana in one sitting.",
+    "valid_targets": [ "self" ],
+    "min_damage": 800,
+    "damage_increment": 15.0,
+    "max_damage": 1200,
+    "max_level": 35,
+    "effect": "recover_energy",
+    "effect_str": "MANA",
+    "shape": "blast",
+    "spell_class": "EATER",
+    "energy_source": "HP",
+    "base_casting_time": 500,
+    "base_energy_cost": 0,
+    "final_energy_cost": 0,
+    "components": "spell_components_dreamdross3",
+    "flags": [ "SOMATIC", "VERBAL", "SILENT", "NO_LEGS" ],
+    "difficulty": 1
+  },
+  {
+    "id": "blood_boil",
+    "type": "SPELL",
+    "name": "Bleed",
+    "description": "With a shouted ancient word, the targets blood temperature begins to increase dangerously.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "blast",
+    "flags": [ "VERBAL", "SOMATIC", "NO_LEGS", "NO_PROJECTILE" ],
+    "energy_source": "MANA",
+    "spell_class": "EATER",
+    "difficulty": 2,
+    "base_casting_time": 100,
+    "base_energy_cost": 150,
+    "damage_type": "biological",
```

```suggestion
    "damage_type": "biological",
```
Is this not heat damage so that the undead aren't affected by it?

---

I thought all heat damage caused a chance to catch on fire?  Which actually would be fine.  Good call.  I'm mostly trying to avoid pure damage as I think it's overused currently.